### PR TITLE
[lab-supervised-learning-sklearn] Duarte Cruz

### PR DESCRIPTION
**Pull request by duarteharris**

### CODE REVIEW

**Comment by ta-data-lis** [comment]
Lab checked. Well done!

> I got 0.8107 and 0.7942, but that's because of the random state seed, right?

In this case, the difference is due to the data used to calculate the score. The first was calculated based on the values of the train set, while the second was calculated based on the values of the test set.

Keep up the good work! 👍
